qujini/settings.py

```diff
@@ -31,13 +31,13 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'questions',
```

Comment:
It's better to leave the orders of stuff as such if there is no solid reason to do that. Maybe, there can be a practice can be established wherein custom apps are written above declaring Django's own apps. These tasks should be usually done by a maintainer, and I advice you to not touch such parts (esp. `settings.py`) of the code unless it is required for implementing what you maybe trying to do.